Kitodo/src/main/java/org/kitodo/services/data/ProcessService.java

```diff
@@ -455,8 +455,20 @@ public void refresh(Process process) {
      * @return list of JSON objects with processes for specific process id
      */
     public List<JSONObject> findByProjectId(Integer id) throws DataException {
-        QueryBuilder query = createSimpleQuery("project", id, true);
-        return searcher.findDocuments(query.toString());
+        return searcher.findDocuments(getQueryProjectId(id).toString());
+    }
+
+    public Long findCountForSortHelperImages(Integer projectId) throws DataException {
+        return findCountAggregation(getQueryProjectId(projectId).toString(), "sortHelperImages");
+    }
+
+    public Double findSumForSortHelperImages(Integer projectId) throws DataException {
```

Comment:
Public method without JavaDoc. Please add the missing JavaDoc!

Reply:
done